sdk/servicebus/azure-servicebus/samples/sync_samples/mgmt_topic.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
```

Comment:
since there's rule/sub/topic it kinda begs the question of both "is queue needed" and "async?"  The former may be worth a mention in the Samples README that it's directly comparable with Topic and avoid replication?   LMK if you have any thoughts @yunhaoling